NOpenCL.Test/TestPlatform.cs

```diff
@@ -48,7 +48,7 @@ private static void TestPlatformProperties(Platform platform)
             }
 
             string version = platform.Version;
-            StringAssert.Matches(version, new Regex("^OpenCL 1.[12] .*$"));
+            StringAssert.Matches(version, new Regex("^OpenCL (?:1.[12]|2.0) .*$"));
```

Comment:
:memo: Originally identified by @SunsetQuest in #1 (https://github.com/tunnelvisionlabs/NOpenCL/pull/1/commits/894262ea9991f6efd3ffca2eeedc3dc536728011).